docs/Reference/Certificates/remoteProvisioning.md

```diff
@@ -0,0 +1,90 @@
+--8<-- "References/abbreviations.md"
+	
+## What is remote provisioning?
+
+Provisioning is the overall process of bringing a remote device under the management of the development system. The process involves activation and configuration of a device without physically touching it. 
+
+!!! Note Whatever 
+    Before provisioning, identify the environment in which the device will be managed. The environments include Public Cloud, Private Cloud, and Hybrid Cloud.
+
+The table below describes the process: 
+
+| Step       | Significance to Provisioning|  
+| :------------------------- | :--------------------- | 
+| 1. Creating a Client Initiated Remote Access (CIRA) configuration | This saved configuration will be used to configure the CIRA protocol for public cloud environments. |
+| 2. Setting up the AMT Profile and choosing an activation type | Choose from two modes, Admin Control Mode (ACM) and Client Control Mode (CCM). If ACM is chosen, add a provisioning certificate in Step 3. CCM does not require a certificate. |
+| 3. Adding a domain profile and associated provisioning certificate (signed or self-signed) | Upload a provisioning certificate. |
```

Comment:
@matt-primrose Is self-signed an option here? Can you act as your own CA and AMT is ok with it?

Reply:
CIRA certificate is always self-signed.  There is an option for a self-signed provisioning certificate, however, it requires someone injecting the root hash into the FW either at the OEM factory or through MEBX.  Not something most folks do.